### ISSUE TITLE
Enable increased map labels size by default when font size is 200%

### ISSUE THREAD

**Issue by RedAuburn** [organicmaps/organicmaps#8319]
Enabling increased map labels size by default when the system font size is 200% (maybe lower?) would make sense.

Is this worth/possible to do on iOS too? @kirylkaveryn

**Comment by kirylkaveryn** [comment]
> Enabling increased map labels size by default when the system font size is 200% (maybe lower?) would make sense.
> 
> 
> 
> Is this worth/possible to do on iOS too? @kirylkaveryn 

I think it possible.
But it requires careful refactoring of the UI elements. Maybe 200% is too big for the all labels and text fields?

**Comment by biodranik** [comment]
Internally, there is a hard-coded multiplier on how fonts should be enlarged in OM:
```
auto constexpr kLargeFontsScaleFactor = 1.6;
```

Maybe it makes sense to remove the explicit "Increase size for map labels" option in OM settings, and always take this multiplier from the system. WDYT?

**Comment by RedAuburn** [comment]
> Maybe it makes sense to remove the explicit "Increase size for map labels" option in OM settings, and always take this multiplier from the system. WDYT?

Seems good, but if scaling is 'normal' it'd be good to still have the option to increase label size, i'd imagine it's useful when mounting a phone to bike handlebars etc.

**Comment by RedAuburn** [comment]
> I think it possible. But it requires careful refactoring of the UI elements. Maybe 200% is too big for the all labels and text fields?

for proper android 14 compliance, it needs to be done 🙃 (https://developer.android.com/about/versions/14/features?hl=en#non-linear-font-scaling)
